src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -15,35 +15,101 @@
+                if (this.displayAsJson)
+                {
+                    System.Console.WriteLine(JsonSerializer.Serialize(activity, this.serializerOptions));
+                }
+                else
+                {
+                    System.Console.WriteLine("Activity ID - " + activity.Id);
+                    if (!string.IsNullOrEmpty(activity.ParentId))
+                    {
+                        System.Console.WriteLine("Activity ParentId - " + activity.ParentId);
+                    }
+
+                    System.Console.WriteLine("Activity OperationName - " + activity.OperationName);
+                    System.Console.WriteLine("Activity DisplayName - " + activity.DisplayName);
+                    System.Console.WriteLine("Activity Kind - " + activity.Kind);
+                    System.Console.WriteLine("Activity StartTime - " + activity.StartTimeUtc);
+                    System.Console.WriteLine("Activity Duration - " + activity.Duration);
+                    if (activity.Tags.Count() > 0)
+                    {
+                        System.Console.WriteLine("Activity Tags");
+                        foreach (var tag in activity.Tags)
+                        {
+                            System.Console.WriteLine($"\t {tag.Key} : {tag.Value}");
+                        }
+                    }
+
+                    if (activity.Events.Any())
+                    {
+                        System.Console.WriteLine("Activity Events");
+                        foreach (var activityEvent in activity.Events)
+                        {
+                            System.Console.WriteLine($"Event Name: {activityEvent.Name} TimeStamp: {activityEvent.Timestamp}");
+                            foreach (var attribute in activityEvent.Attributes)
+                            {
+                                System.Console.WriteLine($"\t {attribute.Key} : {attribute.Value}");
+                            }
+                        }
+                    }
+
+                    if (activity.Baggage.Any())
+                    {
+                        System.Console.WriteLine("Activity Baggage");
+                        foreach (var baggage in activity.Baggage)
+                        {
+                            System.Console.WriteLine($"\t {baggage.Key} : {baggage.Value}");
+                        }
+                    }
+
+                    var resource = activity.GetResource();
+                    if (resource != Resource.Empty)
+                    {
+                        System.Console.WriteLine("Resource associated with Activity");
+                        foreach (var resourceAttribute in resource.Attributes)
+                        {
+                            System.Console.WriteLine($"\t {resourceAttribute.Key} : {resourceAttribute.Value}");
+                        }
+                    }
+
+                    System.Console.WriteLine("\n");
```

Comment:
Curious, do we prefer "\n" or [`System.Environment.NewLine`](https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline) or two `System.Console.WriteLine()`?

Reply:
I don't have preference, but i'd avoid the "\n".